### PR TITLE
Changes for first ntuple production

### PR DESCRIPTION
**Pull request by harrypuuter**

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -565,15 +569,15 @@ def build_config(
...
                     "mc_trigger_sf": "Trg_IsoMu24_pt_eta_bins",
                 },
                 {
-                    "flagname": "trg_wgt_IsoMu27",
+                    "flagname": "trg_wgt_single_mu27",
                     "mc_trigger_sf": "Trg_IsoMu27_pt_eta_bins",
                 },
                 {
-                    "flagname": "trg_wgt_IsoMu24OrIsoMu27",
+                    "flagname": "trg_wgtsingle_mu24ormu27",
```
Comment (nshadskiy) [inline config.py:580]: typo: _single

File: producers/pairquantities.py [hunk 2]
```diff
@@ -273,7 +273,7 @@
     name="gen_match_1",
```
Comment (nshadskiy) [inline producers/pairquantities.py:273]: "taugen_match_1" to match "taugen_match_2" further below

File: quantities/output.py [hunk 3]
```diff
@@ -62,6 +62,8 @@
 decaymode_2 = Quantity("decaymode_2")
 gen_match_1 = Quantity("gen_match_1")
 gen_match_2 = Quantity("gen_match_2")
+tau_gen_match_1 = Quantity("tau_gen_match_1")
+tau_gen_match_2 = Quantity("tau_gen_match_2")
```
Comment (nshadskiy) [inline quantities/output.py:66]: Are still both needed?

File: config.py [hunk 4]
```diff
@@ -51,24 +51,28 @@ def build_config(
...
         {
-            "RunLumiEventFilter_Quantities": ["event", "luminosityBlock"],
-            "RunLumiEventFilter_Quantity_Types": ["ULong64_t", "UInt_t"],
-            "RunLumiEventFilter_Selections": ["3", "318"],
             "PU_reweighting_file": EraModifier(
                 {
-                    "2016": "data/pileup/Data_Pileup_2016_271036-284044_13TeVMoriond17_23Sep2016ReReco_69p2mbMinBiasXS.root",
-                    "2017": "data/pileup/Data_Pileup_2017_294927-306462_13TeVSummer17_PromptReco_69p2mbMinBiasXS.root",
-                    "2018": "data/pileup/Data_Pileup_2018_314472-325175_13TeV_17SeptEarlyReReco2018ABC_PromptEraD_Collisions18.root",
+                    "2016": "",
```
[the remark this answers is not in the document]
Comment (harrypuuter) [reply config.py:56]: due to the era splitting i still left it out

File: quantities/output.py (hunk 3 above)
Comment (harrypuuter) [reply quantities/output.py:66]: `gen_match` and `tau_gen_match` are two different things. The second one is for tauSFs, the first one is for the analysis

Comment (harrypuuter) [reply quantities/output.py:66]: therefore we need both of them

File: config.py (hunk 1 above)
Comment (harrypuuter) [reply config.py]: fixed

**Review by nshadskiy — APPROVED** [review]
Looks good to me!

**Pull request merged**